cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -67,6 +67,9 @@
     
     private final EventBus eventBus = new EventBus();
 
+    // is _bulk_get endpoint supported?
+    private boolean bulkSupported = false;
```

Comment:
I'd probably name this something like `useBulkGet` to indicate that we're deciding a policy.